### PR TITLE
Master

### PR DESCRIPTION
**Pull request by gplarsen**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@gplarsen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: articles/app-service-logic-connector-db2.md [hunk 1]
```diff
@@ -0,0 +1,303 @@
+<properties
+   pageTitle="Using the Microsoft API App Connector for DB2 in a Microsoft Azure App Service Logic App"
+   description="How to use the DB2 Connector with Logic App triggers and actions"   services="app-service\logic"
+   documentationCenter=".net,nodejs,java"
+   authors="plarsen"
```
Comment (davidwrede) [inline articles/app-service-logic-connector-db2.md]: Hey Paul, can you change this to your github username? It looks like it is "gplarsen". 

Thanks,
Dave

**Comment by davidwrede** [comment]
Hey Paul,

All internal FTEs should submit their PRs in the private version of this repo. Let's sync up on doing that.

Thanks,
Dave

**Comment by azurecla** [comment]
__@gplarsen__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

**Pull request closed without merging**